docs/zh/sql-reference/sql-functions/table-functions/files.md

```diff
@@ -204,21 +190,25 @@ StarRocks 当前仅支持通过简单认证访问 HDFS 集群，通过 IAM User
 
 -->
 
-### unload_data
+### unload_data_param
 
 从 v3.2 版本开始，FILES() 支持在远程存储中定义可写入文件以进行数据导出。有关详细说明，请参阅[使用 INSERT INTO FILES 导出数据](../../../unloading/unload_using_insert_into_files.md)。
 
-- `compression`（必填）：导出数据时要使用的压缩方法。有效值：
-  - `uncompressed`：不使用任何压缩算法。
-  - `gzip`：使用 gzip 压缩算法。
-  - `brotli`：使用 Brotli 压缩算法。
-  - `zstd`：使用 Zstd 压缩算法。
-  - `lz4`：使用 LZ4 压缩算法。
-- `max_file_size`：当数据导出为多个文件时，每个数据文件的最大大小。默认值：`1GB`。单位：B、KB、MB、GB、TB 和 PB。
-- `partition_by`：用于将数据文件分区到不同存储路径的列，可以指定多个列。FILES() 提取指定列的 Key/Value 信息，并将数据文件存储在以对应 Key/Value 区分的子路径下。详细使用方法请见以下示例五。
-- `single`：是否将数据导出到单个文件中。有效值：
-  - `true`：数据存储在单个数据文件中。
-  - `false`：如果达到 `max_file_size`，则数据存储在多个文件中。
+```sql
+-- 自 v3.2 版本起支持。
+unload_data_param::=
+    "compression" = "<compression_method>",
+    "max_file_size" = "<file_size>",
+    "partition_by" = "<column_name> [, ...]" 
+    "single" = { "true" | "false" } 
+```
+
+| **参数**          | **必填** | **说明**                                                          |
+| ---------------- | ------------ | ------------------------------------------------------------ |
+| compression      | 是          | 导出数据时要使用的压缩方法。有效值：<ul><li>`uncompressed`：不使用任何压缩算法。</li><li>`gzip`：使用 gzip 压缩算法。</li><li>`brotli`：使用 Brotli 压缩算法。</li><li>`zstd`：使用 Zstd 压缩算法。</li><li>`lz4`: Use the LZ4 compression algorithm.`lz4`：使用 LZ4 压缩算法。</li></ul>                  |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/288e2360-e5e3-4686-b081-c6916b231f81)
The English sentence should be removed.